### PR TITLE
Fix spelling mistake

### PR DESCRIPTION
**Pull request by bleach**

### CODE REVIEW

File: service-manual/operations/service-management.md [hunk 1]
```diff
@@ -209,6 +209,6 @@ performance data, changes to best practice and service demand
 
 * [Functions in service management](http://www.slideshare.net/nuwulang/functions-in-service-operation)
 * [ITIL on Wikipedia](https://en.wikipedia.org/wiki/ITIL)
-* [Introduction to ILIL v3](http://www.best-management-practice.com/gempdf/itsmf_an_introductory_overview_of_itil_v3.pdf)
+* [Introduction to ITIT v3](http://www.best-management-practice.com/gempdf/itsmf_an_introductory_overview_of_itil_v3.pdf)
```
Comment (alexmuller) [inline service-manual/operations/service-management.md]: I think this should be one T and one L :)

Comment (bleach) [inline service-manual/operations/service-management.md]: Ta. That was entirely accidental, honestly :)

**Comment by bleach** [comment]
This should now fix a typo without introduing a new one :/

**Comment by alexmuller** [comment]
:+1:

**Pull request merged**